src/ActiveLogin.Authentication.BankId.AspNetCore.UAParser/UAParserDeviceDetector.cs

```diff
@@ -0,0 +1,156 @@
+using System;
+using ActiveLogin.Authentication.BankId.AspNetCore.SupportedDevice;
+using UAParser;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.UAParser
+{
+    public class UAParserDeviceDetector : IBankIdSupportedDeviceDetector
```

Comment:
As this is a public class, we can add a small XML comment saying what it does.

Reply:
Added XML comment.